packages/client/hmi-client/src/workflow/ops/document/tera-document-operation-node.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+	<main v-if="!fetchingDocument">
+		<template v-if="document">
+			<div>{{ document?.name }}</div>
+			<Button label="Open document" @click="emit('open-drilldown')" severity="secondary" outlined />
+		</template>
+		<template v-else>
+			<Dropdown
+				class="w-full p-dropdown-sm"
+				:options="documents"
+				option-label="name"
+				:model-value="document"
+				placeholder="Select document"
+				@update:model-value="onDocumentChange"
+			/>
+			<tera-operator-placeholder :operation-type="node.operationType" />
```

Comment:
Could you made it available when the document is loaded while we figure out the image placeholder solution?

Reply:
yes this has been added